api/usecases/getAnimeById.go

```diff
@@ -0,0 +1,30 @@
+package usecases
+
+import (
+	animeI "bootCampApi/api/interfaces"
+	"encoding/json"
+	"log"
+	"net/http"
+	"strings"
+)
+
+func GetAnimeById(id string) animeI.AnimeStruct {
+	url := "https://api.jikan.moe/v3/anime/" + id
+	resp, err := http.Get(url)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	var result map[string]interface{}
+	json.NewDecoder(resp.Body).Decode(&result)
+	animeId, _ := result["mal_id"].(float64)
```

Comment:
Acronyms should be all uppercase.
```suggestion
	animeID, _ := result["mal_id"].(float64)
```